cmd/cozypkg/cmd/tap.go

```
@@ -0,0 +1,410 @@
+		k8sClient, err := newClusterClient(tapCmdFlags.kubeconfig)
+		if err != nil {
+			return err
+		}
+
+		// Only roll back objects this invocation actually created; a re-tap is
+		// idempotent and must not delete resources that pre-existed it.
+		created := make([]client.Object, 0, len(toApply))
+		patchOptions := []client.PatchOption{client.FieldOwner("cozypkg"), client.ForceOwnership}
+		for _, obj := range toApply {
+			preExisted := objectExists(ctx, k8sClient, obj)
+			if err := k8sClient.Patch(ctx, obj, client.Apply, patchOptions...); err != nil {
```

golangci-lint (2.12.2)

[error] SA1019: client.Apply is deprecated: Use client.Client.Apply() and client.Client.SubResource("subrsource").Apply() instead. 

(staticcheck)